### ISSUE TITLE
Make Celsius/Fahrenheit switchable

### ISSUE THREAD

**Issue by peternewman** [k4kfh/casparcg-html-templates#3]
Probably also display units on the screen

**Comment by k4kfh** [comment]
In the States (or at least in the southern US where I am at) we _never_ display weather data in Celsius. Don't ask me why, we just don't. So for my use case, I at least need the default to be Fahrenheit, with a simple degree symbol.

In the future, once I get this back up and running again with a proper data source, I might include units as part of the "options" parameters I mentioned in the other thread, but for right now, this is sort of low on the priority list. If you want to implement something, I would suggest waiting a couple weeks, as I am going to try to overhaul the GUI a bit to handle different resolutions better (I've got one school that operates in 1080p and one that operates in 720p).

**Comment by peternewman** [comment]
> In the States (or at least in the southern US where I am at) we _never_ display weather data in Celsius. Don't ask me why, we just don't. So for my use case, I at least need the default to be Fahrenheit, with a simple degree symbol.

So just the degree symbol and no F?

> If you want to implement something, I would suggest waiting a couple weeks, as I am going to try to overhaul the GUI a bit to handle different resolutions better (I've got one school that operates in 1080p and one that operates in 720p).

I was only going to do it via the CSS after properties it's already being done by, which I assume shouldn't impact on your GUI overhaul much/at all?

**Comment by k4kfh** [comment]
Yes, just the degree symbol and no F. If you want to give it a shot, go for it, like I said my development cycle will be sort of slow since I am at Uni right now.

**Comment by peternewman** [comment]
I guess we should add Kelvin too for completeness :stuck_out_tongue:
